### DISCUSSION TITLE
How to configure External JVBs

### DISCUSSION THREAD

**Discussion by tsentekin** [igniterealtime/openfire-pade-plugin#375] [Q&A]
Hi,
The latest Openfire/Pade plugin states that, we are now able to configure external Jvbs for scalability.
Appreciate your help for configuring this option, I can't find any options to configure in Pade plugin,
it'll be great if you can share the configuration steps so I'd like to try this new capability.
Thanks

**Comment by deleolajide** [comment]
> The latest Openfire/Pade plugin states that, we are now able to configure external Jvbs for scalability

![image](https://user-images.githubusercontent.com/110731/146156678-71e1ad4a-7fb1-4e71-a32a-0d834a5a71d1.png)

See this discussion - https://github.com/igniterealtime/openfire-pade-plugin/issues/366

**Comment by gjaekel** [comment]
Year, esp. you may get hints from my suggestions at https://github.com/igniterealtime/openfire-pade-plugin/issues/366#issuecomment-917408513 for further tuning and tweeking. If you have questions, you may discuss it with me (and others here).
